src/renderer/pages/project/script_editor/beat_style.vue

```
@@ -47,5 +72,13 @@
   emit("updateImageNames", imageKey, value);
 };
 
+const openEvent = (event) => {
+  if (event) {
+    emit("update", "imageParams", props.imageParams ?? IMAGE_PARAMS_DEFAULT_VALUES);
+  } else {
+    emit("update", "imageParams", undefined);
+  }
+};
+
 const open = ref(false);
```

ESLint

[error] 'open' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)